shopfloor/services/zone_picking.py

```diff
@@ -477,7 +477,11 @@ def _scan_source_location(self, barcode, confirmation=False):
             match_user=True,
         )
         if move_lines:
+            move_line = first(move_lines)
             response = self._response_for_set_line_destination(first(move_lines))
+            response["data"]["set_line_destination"]["move_line"][
```

Comment:
this is quite ugly. Please find a better way (eg: pass to all `self._response_for*` a `data_for_move_line` optional arg to extend update the value returned by `self._data_for_move_line`

Reply:
done